scripts/webpack/webpack.common.js

```diff
@@ -65,10 +65,6 @@ module.exports = {
         __dirname,
         '../../public/app/redux/hooks'
       ),
-      '@webapp/redux/reducers/tenant': path.resolve(
```

Comment:
But why do we have this here?
```js
      '@webapp/redux/store': path.resolve(
        __dirname,
        '../../public/app/redux/store'
      ),
```

Reply:
Because of this https://github.com/search?q=repo%3Agrafana%2Fpyroscope%20%40webapp%2Fredux%2Fstore&type=code

Reply:
Got it. Thanks!